proto/quicksilver/interchainstaking/v1/query.proto

```diff
@@ -170,6 +175,11 @@ message QueryWithdrawalRecordsResponse {
   cosmos.base.query.v1beta1.PageResponse pagination = 2;
 }
 
+message QueryUserWithdrawalRecordsRequest {
+  string user_address = 1 [(gogoproto.moretags) = "cosmos.AddressString"];
+  cosmos.base.query.v1beta1.PageRequest pagination = 3;
```

Comment:
3?

Reply:
Sorry must have missed this, fixed it now